metplus/wrappers/cyclone_plotter_wrapper.py

```diff
@@ -22,11 +22,10 @@
 import cartopy.crs as ccrs
 import cartopy.feature as cfeature
 from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
-import met_util as util
 import produtil.setup
-import config_metplus
-from command_builder import CommandBuilder
 
+from ..util import met_util as util
+from . import CommandBuilder
```

Comment:
Do you not need a config_metplus here or was it forgotten?

Reply:
'import config_metplus' was in there before, but I searched and it wasn't used anywhere in the code.